docs/products/influxdb/getting-started.rst

```diff
@@ -0,0 +1,48 @@
+Getting started with Aiven for InfluxDB®
+########################################
+
+Aiven services are managed from Aiven `web console <https://console.aiven.io/>`_. First login to the console with your email address and password and you will be automatically taken to the "Services" view that shows all the services of the currently selected project.
+
+**Projects** allow organizing groups of services under different topics and each project can for example have different billing settings. An empty project is created for you automatically when you sign-up and the free credits are attached to this project. You can create new projects by clicking the project name in the left side-bar and selecting "**Create a new project**". The same menu can also be used to switch between projects.

```

Comment:
I believe we can remove this section since it's not really linked to Influx only and is explained elsewhere?